components/match2/wikis/ageofempires/player_ext_custom.lua

```diff
@@ -0,0 +1,94 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:Player/Ext/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local PlayerExt = require('Module:Player/Ext')
+local Table = require('Module:Table')
+local Array = require('Module:Array')
+local TeamTemplate = require('Module:TeamTemplate')
+local TournamentUtil = require('Module:Tournament/Util')
+local Json = require('Module:Json')
+local PageVariableNamespace = require('Module:PageVariableNamespace')
+local playerVars = PageVariableNamespace({namespace = 'Player', cached = true})
+
+local CustomPlayerExt = Table.deepCopy(PlayerExt)
+
+function CustomPlayerExt.fetchTeamHistoryEntry(resolvedPageName, date)
+	date = date or TournamentUtil.getContextualDateOrNow()
+
+	local conditions = {
+		'[[type::teamhistory]]',
+		'[[pagename::' .. resolvedPageName:gsub(' ', '_') .. ']]',
+		'([[extradata_joindate::<' .. date .. ']] or [[extradata_joindate::' .. date .. ']])',
+		'[[extradata_joindate::>]]',
+		'[[extradata_leavedate::>' .. date .. ']]',
+	}
+	local records = mw.ext.LiquipediaDB.lpdb('datapoint', {
+		conditions = table.concat(conditions, ' and '),
+		query = 'information, extradata',
+	})
```

Comment:
This needs to be sorted on date unless I'm thinking incorrect
Also limit = 1 should be set as well.

Reply:
This is mostly copied from regular `Module:Player/Ext`, so might need to be fixed there as well.

Reply:
Not sure why this would have to be sorted by date (and which), at least on AoE the actualy `date` field isn't even set for these datapoints.
Added the limit now.

Reply:
I was thinking wrong